docs/content/2.storage-drivers/s3.md

```diff
@@ -79,36 +74,20 @@ volumes:
 
 Don't forget to set the `STORAGE_DRIVER` environment variable to `s3` to use the S3 storage driver.
 
+The AWS SDK will automatically use any AWS credentials available in the environment, e.g. `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_REGION`.
+
 #### `STORAGE_S3_BUCKET`
 
 Example: `gh-actions-cache`
 
 The name of the S3 bucket used for storage.
 
-#### `STORAGE_S3_ACCESS_KEY`
-
-Example: `access_key`
-
-The access key for S3 storage.
-
-#### `STORAGE_S3_SECRET_KEY`
-
-Example: `secret_key`
-
-The secret key for S3 storage.
-
```

Comment:
I would keep the manual approach in the docs, as not everyone using S3 is running with AWS.